wiki/Replay/en.md

```diff
@@ -0,0 +1,285 @@
+
+To filter the comments by difficulty, enable `Difficulty (#)`.
+To filter the comments by the overall beatmapset, enable `Song (#)`.
+When watching a top play from the _Global leaderboard_, enable `Player (#)` to show comments from the player itself.
+
+Mapper's comment will be placed above the flowing comments with a light-blue text and an jump-out/jump-in animation instead.
+Depending on resolution, it may be obstructed by a **UNRANKED** notice when watching an Auto replay.
+To remove the **UNRANKED** notice, either watch a top replay from the _Global leaderboard_, or a local replay from the _Local leaderboard_ without the Auto mod used.
+
+When watching a top replay, the player of the replay's comment(s) will be shown _below_ the screen instead, and will only be shown in this replay only.
+
+By default without active osu!supporter, comments will be coloured based on chat colour (normal accounts will be white) except for mapper's comment(s) (light-blue). 
+
+### Collections
+
+Add this beatmap(set) to _Collections_.
+This will open up the Collections screen to put the current beatmap(set) into specific sections or rename the sections.
+
+The replay will not pause and continue at the background while in this screen.
+
+### Playback Speed
+
+Click to adjust the `1x`/`2x`/`0.5x` playback speed of the replay to `2x`/`0.5x`/`1x` playback speed.
+Shortcut key is the `F` key.
+
+### End Replay
+
+Self-explanatory; click to end the replay.
+Shortcut key is the `Esc` key.
+
+## Types of Replay
+
+_Full documentation regarding the .osr file format (for developers): [.osr file format][osr wikilink]_
+
+To export a replay, press `F2` in the results screen.
+Exported replay will have an `.osr` file extension with a clean filename format as shown below:-
+
+```
+Format : {LocalPlayerName} - {Artist} - {Title} [{Difficulty}]({YYYY-MM-DD}) {GameMode}
+Example: dummytest1 - Loituma - Ievan Polkka [SPINNER-MADNESS] (2013-08-12) OsuMania
+```
+
+Do note that exported replay will **not work** if the **difficulty/beatmap linked to the replay file was missing/not found**.
+Once opened, exported replay will be added to the local leaderboard and to the backend (specifically, the hidden folder, `Data\r`).
+The original exported replay will not be lost after opening.
+
+A quick way to remember whether the replay will be saved internally is the same as whether it will be saved in the local leaderboard after passing the difficulty.
+
+A white, hovering text from right-to-left will be shown on top but below the **UNRANKED** notice when watching a replay.
+
+The format is shown below unless stated otherwise:-
+
+```
+Format : REPLAY MODE - Watching {AccountName} play {ArtistName} - {BeatmapsetName} [{Difficulty}]
+Example: REPLAY MODE - Watching osu! play Peter Lambert - osu!tutorial [Gameplay Basics]
+```
+
+### Local (Solo)
+
+![A local replay][Replay Solo]
+
+_A typical local replay_
+
+Solo local plays regardless of connection status or logged-in account name.
```

Comment:
Change to: `You can play Solo Mode regardless of your internet connection or log-in status.`